src/dialogs/more-info/components/siren/ha-more-info-siren-advanced-controls.ts

```diff
@@ -0,0 +1,223 @@
+          ></ha-icon-button>
+          <span slot="title" id="dialog-light-color-favorite-title"
+            >${this.hass.localize(
+              "ui.components.siren.advanced_controls"
+            )}</span
+          >
+        </ha-dialog-header>
+        <div slot="content">
+          <div class="options">
+            ${supportsTones
+              ? html`
+                  <ha-select
+                    .label=${this.hass.localize("ui.components.siren.tone")}
+                    @closed=${stopPropagation}
+                    @change=${this._handleToneChange}
+                    .value=${this._tone}
+                  >
+                    ${Object.entries(
+                      this._stateObj.attributes.available_tones
+                    ).map(
+                      ([toneId, toneName]) => html`
+                        <ha-list-item .value=${toneId}
+                          >${toneName}</ha-list-item
+                        >
+                      `
+                    )}
+                  </ha-select>
+                `
+              : nothing}
+            ${supportsVolume
+              ? html`
+                  <ha-textfield
+                    type="number"
+                    .label=${this.hass.localize("ui.components.siren.volume")}
+                    .suffix=${"%"}
+                    .value=${this._volume ? this._volume / 100 : undefined}
+                    @change=${this._handleVolumeChange}
+                    .min=${0}
+                    .max=${100}
+                    .step=${1}
+                  ></ha-textfield>
+                `
+              : nothing}
+            ${supportsDuration
+              ? html`
+                  <ha-textfield
+                    type="number"
+                    .label=${this.hass.localize("ui.components.siren.duration")}
+                    .value=${this._duration}
+                    @change=${this._handleDurationChange}
+                  ></ha-textfield>
+                `
+              : nothing}
+          </div>
+          <div class="controls">
+            <ha-control-button
+              .label=${this.hass.localize("ui.card.common.turn_on")}
+              @click=${this._turnOn}
+            >
+              <ha-svg-icon .path=${mdiPlay}></ha-svg-icon>
+            </ha-control-button>
+            <ha-control-button
+              .label=${this.hass.localize("ui.card.common.turn_off")}
+              @click=${this._turnOff}
+            >
+              <ha-svg-icon .path=${mdiStop}></ha-svg-icon>
+            </ha-control-button>
+          </div>
+        </div>
+        <div slot="actions">
+          <ha-button @click=${this.closeDialog}>
+            ${this.hass.localize("ui.common.close")}
+          </ha-button>
+        </div>
+      </ha-md-dialog>
+    `;
+  }
+
+  private _handleToneChange(ev) {
+    this._tone = ev.target.value;
+  }
+
+  private _handleVolumeChange(ev) {
+    this._volume = parseFloat(ev.target.value) * 100;
```

Comment:
```suggestion
    this._volume = parseFloat(ev.target.value) / 100;
```

I think 1% vol should be the value 0.01 instead of 100 right?